### PR TITLE
Add final stdlib dependencies to distribution

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

This PR adds following std libs,
1. Java.arrays
2. Mime
3. Email
4. Http
5. Grpc

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -36,3 +36,8 @@ stdlibAuthVersion=1.0.0-SNAPSHOT
 stdlibOAuth2Version=1.0.0-SNAPSHOT
 stdlibLdapVersion=1.0.0-SNAPSHOT
 stdlibCryptoVersion=1.0.0-SNAPSHOT
+stdlibJavaarraysVersion=0.9.0-SNAPSHOT
```
Comment (ldclakmal) [inline gradle.properties:39]: Shall we rename as follows?
```suggestion
stdlibJavaArraysVersion=0.9.0-SNAPSHOT
```

**Review by ldclakmal — APPROVED** [review]
Added a suggestion. Rest LGTM.

**Review by BuddhiWathsala — APPROVED** [review]
LGTM

**Review by daneshk — APPROVED** [review]

**Comment by BuddhiWathsala** [comment]
Kafka BBEs are failing, I guess it couldn't find avro jar. @ThisaruGuruge don't we need to pack that avro jar. https://github.com/ballerina-platform/module-ballerinax-kafka/blob/c3afb677ef301421b0cdd26e18e7b5a602099c86/kafka-ballerina/build.gradle#L39

**Comment by niveathika** [comment]
@BuddhiWathsala I had an offline discussion with @ThisaruGuruge it seems Avro dependency needs to be added by the user explicitly. We will be ignoring Kafka BBEs from build meanwhile.

File: gradle.properties (hunk 1 above)
Comment (niveathika) [reply gradle.properties]: +1. Fixed in 0e96c07

**Pull request merged**